packages/model-registry/upstream/frontend/src/app/pages/modelRegistry/screens/ModelVersionDetails/ModelVersionDetails.tsx

```diff
@@ -142,4 +145,19 @@ const ModelVersionsDetails: React.FC<ModelVersionsDetailProps> = ({ tab, ...page
   );
 };
 
+const ModelVersionsDetails: React.FC<ModelVersionsDetailProps> = (props) => {
+  const { preferredModelRegistry } = React.useContext(ModelRegistrySelectorContext);
+  const { modelVersionId: mvId } = useParams();
+  
+  const labelSelectors = React.useMemo(() => {
+    return mvId ? { [KnownLabels.MODEL_VERSION_ID]: mvId } : undefined;
+  }, [mvId]);
```

Comment:
Don't we also need to filter on KnownLabels.REGISTERED_MODEL_ID here in the labelSelectors?